client/me/get-apps/desktop-download-card.jsx

```diff
@@ -0,0 +1,123 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { translate } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Card from 'components/card';
+import Button from 'components/button';
+
+const WINDOWS_LINK = 'https://apps.wordpress.com/d/windows?ref=getapps';
+const MAC_LINK = 'https://apps.wordpress.com/d/osx?ref=getapps';
+const LINUX_TAR_LINK = 'https://apps.wordpress.com/d/linux?ref=getapps';
+const LINUX_DEB_LINK = 'https://apps.wordpress.com/d/linux-deb?ref=getapps';
+
+const TrackAppDownloadClick = () => {};
+
+const GetLinkAnchorTag = ( platformLink ) => {
+	return <a href={ platformLink } onClick={ TrackAppDownloadClick } />;
+};
+
+const getButtonLink = ( platform ) => {
+	switch ( platform ) {
+		case 'MacIntel':
+			return MAC_LINK;
+		case 'Linux i686':
+		case 'Linux i686 on x86_64':
+			return LINUX_TAR_LINK;
+	}
+};
+
+const getCardTitle = ( platform ) => {
+	switch ( platform ) {
+		case 'MacIntel':
+			return translate( 'Desktop App for Mac' );
```

Comment:
Hi! I've found a possible matching string that has already been translated **7** times:
`translate( 'Desktop App' )` *ES Score: 7*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).